app/routes/error.js

```diff
@@ -6,32 +6,28 @@ const logger = require('winston');
 function Route(app) {
   logger.log('-AddRoute: error');
   app.use((error, req, res, next) => {
-    // treat as 404
     const msg = error.message;
+
+    // If error message looks like 404, treat like 404
     if (msg && (msg.indexOf('not found') >= 0 || msg.indexOf('Cast to ObjectId failed') >= 0)) {
       return next();
     }
 
-    logger.error(error.stack);
+    // Error was expected and most likely the clients fault
+    if (error.statusCode) {
```

Comment:
I would put this above previous statement. if statusCode is known ots not needed to check any other thing..